libs/base/configkeys.h

```diff
@@ -95,6 +95,9 @@
 #define CFG_PIN_PWREN 77
 #define CFG_DISPLAY_TYPE 78
 
+#define CFG_ROTARY_ENCODER_PIN_A 79
+#define CFG_ROTARY_ENCODER_PIN_B 80
```

Comment:
actually I think this should be `CFG_PIN_ROTARY_ENCODER_A` so we can use `PIN(ROTARY_ENCODER_A)`